apps/ledger-live-desktop/src/renderer/screens/account/index.jsx

```diff
@@ -39,6 +39,10 @@ import EmptyStateAccount from "./EmptyStateAccount";
 import TokensList from "./TokensList";
 import CompoundBodyHeader from "~/renderer/screens/lend/Account/AccountBodyHeader";
 import useCompoundAccountEnabled from "~/renderer/screens/lend/useCompoundAccountEnabled";
+import { getBannerProps, AccountBanner } from "./AccountBanner";
+
+import { useDispatch } from "react-redux";
```

Comment:
those two imports use libraries that are already called a bit earlier in the file. Could you group them ?
`react-redux` and `react-i18next`

Reply:
Done